spec/requests/api/merge_requests_spec.rb

```diff
@@ -364,6 +385,15 @@
       json_response.first['author']['id'].should == user.id
     end
 
+    it "should return comments of merge_request by iid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.